pg_search/sql/pg_search--0.12.2--0.13.0.sql

```diff
@@ -50,3 +50,17 @@ CREATE  FUNCTION "index_fields"(
 STRICT
 LANGUAGE c /* Rust */
 AS 'MODULE_PATHNAME', 'index_fields_wrapper';
+
+/* </end connected objects> */
+/* <begin connected objects> */
+-- pg_search/src/api/index.rs:552
+-- pg_search::api::index::regex_phrase
+CREATE  FUNCTION "regex_phrase"(
+	"field" FieldName, /* pg_search::api::index::FieldName */
+	"regexes" TEXT[], /* alloc::vec::Vec<alloc::string::String> */
+	"slop" INT DEFAULT NULL, /* core::option::Option<i32> */
+	"max_expansions" INT DEFAULT NULL /* core::option::Option<i32> */
+) RETURNS SearchQueryInput /* pg_search::query::SearchQueryInput */
+IMMUTABLE PARALLEL SAFE
+LANGUAGE c /* Rust */
+AS 'MODULE_PATHNAME', 'regex_phrase_wrapper';
```

Comment:
added schema upgrade

Reply:
@Weijun-H This needs to be added to a new file called `pg_search--0.13.0--0.13.1.sql` since we've already realised `v0.13.0`.

Reply:
I moved to `pg_search--0.13.0--0.13.1.sql` but still failed.

Reply:
yeah, the schemabot doesn't know about the upgrade file, so that failure is okay.